lib/contestant.rb

```diff
@@ -0,0 +1,26 @@
+class Contestant
+  attr_reader :first_name, :last_name, :age, :state_of_residence, :spending_money, :game_interests
+
+  def initialize(attribute)
+    @first_name         = attribute[:first_name]
+    @last_name          = attribute[:last_name]
+    @age                = attribute[:age]
+    @state_of_residence = attribute[:state_of_residence]
+    @spending_money     = attribute[:spending_money]
+    @out_of_state       = false
+    @game_interests     = []
+  end
+
+  def full_name
+    "#{first_name} ".concat(last_name)
```

Comment:
Good use of `concat`; you could also do something like `"#{first_name} #{last_name}"` or `first_name + ' ' + last_name`